lib/mozart_fetcher/http_client.ex

```diff
@@ -1,13 +1,19 @@
 defmodule HTTPClient do
   use ExMetrics
 
+  @content_timeout MozartFetcher.content_timeout()
+
   def get(endpoint) do
     ExMetrics.timeframe "function.timing.http_client.get" do
-      cert = Application.get_env(:mozart_fetcher, :dev_cert_pem)
-      timeout = Application.get_env(:mozart_fetcher, :timeout)
+      cert = MozartFetcher.cert()
```

Comment:
I haven't moved this to an `@cert` attribute, but I think we could. If we did this I believe we would specify the cert file at `mix release` time in the fetcher build repo, and not in the cosmos config. Using the `@cert` might be very slightly more performant?